### PR TITLE
Add Kcaldav

### PR DESCRIPTION
**Pull request by ligurio**
Thank you for taking the time to work on a PR for Awesome-selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by nodiscc** [comment]
Please merge https://github.com/ligurio/awesome-selfhosted/pull/1

**Comment by n8225** [comment]
Please add license and language according to formatting guidelines, example: \`C\` \`ISC\`
Please add source code as well.

**Comment by nodiscc** [comment]
I've closed https://github.com/ligurio/awesome-selfhosted/pull/1due to merge conflicts

@ligurio Please remove the double space between source code and license.

Is it ok to point the source code link to https://github.com/kristapsdz/kcaldav? (better as it's under version control, whereas tarballs are not)

**Comment by ligurio** [comment]
Done

File: README.md [hunk 1]
```diff
@@ -171,6 +171,7 @@ Some [Groupware](#groupware) solutions also feature calendar/address book editin
   * [CalendarServer](http://trac.calendarserver.org/) - A standards-compliant server implementing the CalDAV and CardDAV protocols. `Apache` `Python`
   * [calypso](http://keithp.com/calypso/) - A python-based CalDAV/CardDAV server. ([Source Code](http://keithp.com/cgi-bin/gitweb.cgi?p=calypso.git;a=tree)) `GPLv3` `Python`
   * [DAViCal](http://www.davical.org/) - A server for calendar sharing (CalDAV). ([Source Code](https://gitlab.com/davical-project/davical)) `MIT` `PHP`
+   * [Kcaldav](https://kristaps.bsd.lv/kcaldav/) - is a simple, safe, and minimal CalDAV server depending only on [kcgi(3)](http://kristaps.bsd.lv/kcgi/), [SQLite](https://www.sqlite.org/), and [expat(3)](http://expat.sourceforge.net/). ([Source code](https://github.com/kristapsdz/kcaldav))  `ISC`   `C`
```
Comment (n8225) [inline README.md]: There should be only 1 space between the source code and the license and only 1 between the license and the language.

Is this a fork of DAVical? If not remove one space before the *.

Comment (n8225) [inline README.md]: There should be only 1 space between the source code and the license and only 1 between the license and the language.

Is this a fork of DAVical? If not remove one space before the *.

File: README.md [hunk 2]
```diff
@@ -171,6 +171,7 @@ Some [Groupware](#groupware) solutions also feature calendar/address book editin
   * [CalendarServer](http://trac.calendarserver.org/) - A standards-compliant server implementing the CalDAV and CardDAV protocols. `Apache` `Python`
   * [calypso](http://keithp.com/calypso/) - A python-based CalDAV/CardDAV server. ([Source Code](http://keithp.com/cgi-bin/gitweb.cgi?p=calypso.git;a=tree)) `GPLv3` `Python`
   * [DAViCal](http://www.davical.org/) - A server for calendar sharing (CalDAV). ([Source Code](https://gitlab.com/davical-project/davical)) `MIT` `PHP`
+  * [Kcaldav](https://kristaps.bsd.lv/kcaldav/) - is a simple, safe, and minimal CalDAV server depending only on [kcgi(3)](http://kristaps.bsd.lv/kcgi/), [SQLite](https://www.sqlite.org/), and [expat(3)](http://expat.sourceforge.net/). ([Source code](https://github.com/kristapsdz/kcaldav)) `ISC`  `C`
```
Comment (n8225) [inline README.md]: There is still an extra space between \`ISC\` and \`C\`

**Comment by n8225** [comment]
Looks good, Thank you.

**Comment by Kickball** [comment]
Was an extra c meant to be added to the end of the viewer in the description of GoAccess?

File: README.md [hunk 3]
```diff
@@ -171,6 +171,7 @@ Some [Groupware](#groupware) solutions also feature calendar/address book editin
   * [CalendarServer](http://trac.calendarserver.org/) - A standards-compliant server implementing the CalDAV and CardDAV protocols. `Apache` `Python`
   * [calypso](http://keithp.com/calypso/) - A python-based CalDAV/CardDAV server. ([Source Code](http://keithp.com/cgi-bin/gitweb.cgi?p=calypso.git;a=tree)) `GPLv3` `Python`
   * [DAViCal](http://www.davical.org/) - A server for calendar sharing (CalDAV). ([Source Code](https://gitlab.com/davical-project/davical)) `MIT` `PHP`
+  * [Kcaldav](https://kristaps.bsd.lv/kcaldav/) - is a simple, safe, and minimal CalDAV server depending only on [kcgi(3)](http://kristaps.bsd.lv/kcgi/), [SQLite](https://www.sqlite.org/), and [expat(3)](http://expat.sourceforge.net/). ([Source code](https://github.com/kristapsdz/kcaldav)) `ISC` `C`
```
Comment (AndyR207) [inline README.md]: [Kcaldav](https://kristaps.bsd.lv/kcaldav/) - A simple, safe, and minimal CalDAV server depending only on [kcgi(3)](http://kristaps.bsd.lv/kcgi/), [SQLite](https://www.sqlite.org/), and [expat(3)](http://expat.sourceforge.net/). ([Source Code](https://github.com/kristapsdz/kcaldav)) `ISC` `C`

**Comment by Kickball** [comment]
@ligurio are you able to work on the suggested changes above?

**Comment by AndyR207** [comment]
@ligurio Please work on the changes requested in this PR, or it will be closed in 3 days.

**Comment by ligurio** [comment]
@AndyR207 feel free to close, I don't have free time for this

**Pull request closed without merging**